logic_mod/Blocks/Api/CpuMath.cs

```diff
@@ -63,12 +83,31 @@ public object Abs(VarCtx ctx, object[] x)
                 throw new Exception("Invalid value");
             return (float)Math.Abs(v);
         }
+        public object Ceiling(VarCtx ctx, object[] x)
+        {
+            if (x.Length < 1 || !BlockUtils.TryGetFloat(x[0], out float v))
+                throw new Exception("Invalid value");
+            return (int)Math.Ceiling(v);

```

Comment:
Do not return "int", it is not supported by this JS implementation. JS expects "long"